src/aiperf/plot/plot_controller.py

```
@@ -0,0 +1,172 @@
+        verbose: bool = False,
+    ):
+        self.paths = paths
+        self.output_dir = output_dir
+        self.mode = mode
+        self.theme = theme
+        self.verbose = verbose
+
+        log_level = "DEBUG" if verbose else "INFO"
+        try:
+            setup_plot_logging(output_dir, log_level=log_level)
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)